OAuth2/OAuth2.csproj

```diff
@@ -4,11 +4,14 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <PackageTags>OAuth2, OAuth, DigitalOcean, Facebook, Foursquare, GitHub, Google, Instagram, LinkedIn, MailRu, Odnoklassniki, Twitter, VK (Vkontakte), Windows Live, Yandex, VSTS</PackageTags>
     <DefineConstants>FRAMEWORK;NETSTANDARD2_0;</DefineConstants>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>OAuth2.pfx</AssemblyOriginatorKeyFile>
+    <DelaySign>false</DelaySign>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
-    <PackageReference Include="RestSharp" Version="106.3.0" />
+    <PackageReference Include="RestSharp" Version="106.2.1" />
```

Comment:
Any reason for downgrading? If anything we should be moving to the latest, can we revert this change